src/api/todos.ts

```diff
@@ -5,4 +5,14 @@ export const getTodos = (userId: number) => {
   return client.get<Todo[]>(`/todos?userId=${userId}`);
 };
 
-// Add more methods here
+export const addTodos = async (data: Partial<Todo>) => {
```

Comment:
When you are creating a new todo - there should be all properties I guess, So don't use `Partial ` here, if there are some fileds you don' wanna have -> use `Omit<Todo, 'removedProperty'>` instead

Reply:
thanks, I know about this type, but I extended the todo interface and now I need to discard already two keys